Assets/Scripts/Effects/AnimateWater.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Effects
+{
+    public class AnimateWater : MonoBehaviour
+    {
+        [SerializeField] [Tooltip("Determines the difference between the peak and bottom of a wave")]
+        private float power = 3f;
```

Comment:
Please assign [SerializedFields] in the inspector or make it a constant and remove [SerializeField]

Reply:
What is the problem with assigning them in the script?
I'd really like to have some default values here.

Reply:
You don't normally assign a field on creation. And it's confusing to have values in the script which are most likely outdated because you set them in the prefab anyway. I think having it the same for every class is best... either inspector + prefab or script.

Reply:
aaallllrighty then